packages/authentication/ios/Plugin/FirebaseAuthentication.swift

```diff
@@ -157,6 +167,27 @@ public typealias AuthStateChangedObserver = () -> Void
         }
     }
 
+    @objc func signInWithEmailLink(_ call: CAPPluginCall) {
+        let email = call.getString("email", "")
+        let emailLink = call.getString("emailLink", "")
+
+        if self.isSignInWithEmailLink(link: emailLink) {
+            self.savedCall = call
+            Auth.auth().signIn(withEmail: email, link: emailLink) { _, error in
```

Comment:
Is there no `authResult` like in line `266`?
If yes, please get the user and any other infos from there.

Example:

```swift
uth.auth().signIn(with: credential) { (authResult, error) in
            if let error = error {
                self.handleFailedSignIn(message: nil, error: error)
                return
            }
            guard let savedCall = self.savedCall else {
                return
            }
            let result = FirebaseAuthenticationHelper.createSignInResult(credential: authResult?.credential, user: authResult?.user, idToken: idToken,
                                                                         nonce: nonce, accessToken: accessToken, additionalUserInfo: authResult?.additionalUserInfo)
            savedCall.resolve(result)
}
```